### PR TITLE
fix(patient-account): fixed date format, add printing declaration. Connected to #10

### PR DESCRIPTION
**Pull request by ivchenkodima**

### CODE REVIEW

File: packages/patient-account/src/components/DeclarationHistory.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+const DeclarationHistory = ({ data }) => (
+  <CabinetTable
+    data={data}
+    header={{
+      start_date: "Дата ухвалення декларації",
+      status: "Статус",
+      division_name: "Назва відділення",
+      employee: (
+        <Fragment>
```
Comment (kiurchv) [inline packages/patient-account/src/components/DeclarationHistory.js]: Take preference to `<>` syntax, please

File: packages/patient-account/src/components/DictionaryValue.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import styled from "react-emotion/macro";
+import { Query } from "react-apollo";
+import { gql } from "graphql.macro";
+
+const DictionaryValue = ({ name, children, render }) => (
+  <Query
+    fetchPolicy="cache-first"
+    context={{ credentials: "same-origin" }}
+    query={gql`
+      query DictionriesQuery {
```
Comment (ephor) [inline packages/patient-account/src/components/DictionaryValue.js]: Diction**a**riesQuery

File: packages/patient-account/src/globalStyles.js [hunk 3]
```diff
@@ -57,9 +57,16 @@ injectGlobal`
 
   body {
     color: #3c4858;
-    font: 400 12px/1.6 "GothamPro", sans-serif;
+    font: 400 12px/1.6 "GothamPro", sans-serif !important;
```
Comment (ephor) [inline packages/patient-account/src/globalStyles.js]: !important? what for?

File: packages/patient-account/src/components/PatientDeclarations.js [hunk 4]
```diff
@@ -2,86 +2,43 @@ import React, { Fragment } from "react";
...
 
-class PatientDeclarations extends React.Component {
-  state = {
-    show: false
-  };
-  render() {
-    const { data } = this.props;
-    const { show } = this.state;
+const PatientDeclarations = ({ data, match }) => {
+  const [active] = data.filter(
```
Comment (kiurchv) [inline packages/patient-account/src/components/PatientDeclarations.js]: In this case `Array.prototype.find` fits better

File: packages/patient-account/src/components/PatientDeclarations.js [hunk 5]
```diff
@@ -2,86 +2,43 @@ import React, { Fragment } from "react";
...
+    ({ status }) => status === "active" || status === "pending_verification"
+  );
+  return (
+    <>
+      <ActiveDeclaration active={active} />
+      <ShowBlock>
+        <Link
+          to={`${match.url === "/" ? "/declarations" : "/"}`}
```
Comment (kiurchv) [inline packages/patient-account/src/components/PatientDeclarations.js]: Template literals are redundant here.

File: packages/patient-account/src/pages/EmployeePage.js [hunk 6]
```diff
@@ -80,12 +63,18 @@ const EmployeePage = ({ match, history }) => (
                     <Text>{item.speciality}</Text>
                     <Text>{item.institution_name}</Text>
                     <Text>
-                      {getDictValue(educationTypes.values, item.degree)} <br />
+                      <DictionaryValue
+                        name="EDUCATION_DEGREE"
+                        render={v => v[item.degree]}
```
Comment (kiurchv) [inline packages/patient-account/src/pages/EmployeePage.js]: We should consider for using `lodash.get` for deep access to dictionary values

File: packages/patient-account/src/components/PatientDeclarations.js (hunk 5 above)
Comment (ivchenkodima) [reply packages/patient-account/src/components/PatientDeclarations.js]: Done

File: packages/patient-account/src/components/DictionaryValue.js (hunk 2 above)
Comment (ivchenkodima) [reply packages/patient-account/src/components/DictionaryValue.js]: Done

File: packages/patient-account/src/components/DeclarationHistory.js (hunk 1 above)
Comment (ivchenkodima) [reply packages/patient-account/src/components/DeclarationHistory.js]: Done

File: packages/patient-account/src/components/PatientDeclarations.js (hunk 4 above)
Comment (ivchenkodima) [reply packages/patient-account/src/components/PatientDeclarations.js]: Done

**Pull request merged**